.env_aws

```diff
@@ -2,8 +2,9 @@
 
 export DATADIR=/data/2018-07-10
 export DATAFILE=labelled_level2.csv.gz
-export SINCE_THRESHOLD=1940-01-01
-export EXPERIMENT_NAME=since_1940.h5
+export SINCE_THRESHOLD=2009-08-14
+export EXPERIMENT_NAME=last_11_years.h5
+export COMET_API_KEY=ohCiHJ1pXoSLP46AeItiYm0ke
```

Comment:
The primary reason for having this as an environment variable is to avoid committing it (which I should have mentioned before), so I'd remove it from here, and add it to the `.local.envrc` file wherever you need it.